src/lib.rs

```diff
@@ -1,7 +1,6 @@
-#![feature(once_cell)] //for synclazy
+#![feature(lazy_cell)]
```

Comment:
Why we replace `oncecell` with `lazycell` here?

Reply:
That's a change due to us needing to update for a newer version of Rust.